stellar/deploy-contract.js

```diff
@@ -25,7 +25,7 @@ const {
 } = ethers;
 require('./cli-utils');
 
-const AXELAR_RELEASE_BASE_URL = 'https://static.axelar.network/releases/axelar-cgp-stellar';
+const AXELAR_RELEASE_BASE_URL = 'https://static.axelar.network/releases/axelar-amplifier-stellar';
```

Comment:
before merging this, we also need to rename it on R2 at the same time to avoid breaking the deployment